### DISCUSSION TITLE
Select virtualization

### DISCUSSION THREAD

**Discussion by skelawsky** [mantinedev/mantine#1594] [Q&A] [answered]
Partly related to mantine. Has anyone created a virtualized select component? Which library did you use? Will react-window work? Tips on how to go about it? Thanks!

**Comment by rtivital** [comment, marked as answer]
I do not think that it is possible to use virtualization libraries with select and other inputs dropdowns, if you have a lot of values (> 100) you should use `limit` prop and make user search for the option.

**Reply by SaeWooKKang** [reply]
It would be nice to add a virtualization for the reasons explained in the following [issue](https://github.com/mantinedev/mantine/issues/1747#issuecomment-1818776547). 

Do you have any plans in the future?
